projects/swarmd/swarmd.yml

```diff
@@ -29,27 +31,16 @@ services:
   - name: ntpd
     image: "linuxkit/openntpd:45deeb05f736162d941c9bf494983f655ab80aa5"
   - name: swarmd
-    image: "linuxkit/swarmd:a2f57f14f07fb6d7cded7832b2dabe878b28554e"
+    image: "linuxkit/swarmd:fa2772d78255d6868bbe8910bcca3cedbb067bf8"
     command: ["/usr/bin/swarmd", "--containerd-addr=/run/containerd/containerd.sock", "--log-level=debug", "--state-dir=/var/lib/swarmd"]
     capabilities:
-     - CAP_CHOWN
-     - CAP_DAC_OVERRIDE
-     - CAP_FSETID
-     - CAP_FOWNER
-     - CAP_MKNOD
-     - CAP_NET_RAW
-     - CAP_SETGID
-     - CAP_SETUID
-     - CAP_SETFCAP
-     - CAP_SETPCAP
-     - CAP_NET_BIND_SERVICE
-     - CAP_SYS_CHROOT
-     - CAP_KILL
-     - CAP_AUDIT_WRITE
+     - all
     pid: host
```

Comment:
The alternative is to bind the namespace to a file and pass that instead, then you no longer need the same `/proc`. I think you just `mount --bind /proc/PID/ns/net /somewhere/to/keep/it` and use that as equivalent. You can also pass filenames to the `net: ...` in the linuxkit config.